gulp_tasks/footer.js

```diff
@@ -5,7 +5,7 @@ const pkg = require( '../package.json' );
 module.exports = {
 	coreIconsVariables() {
 		return gulp.src( `${ pkg._core_theme_pcss_path }utilities/variables/_icons.pcss` )
-			.pipe( footer( '}' ) )
+			.pipe( footer( '}\n' ) )
```

Comment:
Just a few tweaks to the icon tasks here and in header, footer, and replace) to better format the copied css files.